frontend/package.json

```diff
@@ -28,9 +29,12 @@
     "react-native-reanimated": "~1.13.0",
     "react-native-safe-area-context": "3.1.4",
     "react-native-screens": "~2.10.1",
-    "react-native-web": "~0.13.7",
+    "react-native-web": "~0.11",
```

Comment:
Keeping version `"~0.11"` rather than `"~0.13.7"` shouldn't be an issue, since I don't think anyone else runs their code on the web. However, I'd recommend using an Android/iPhone emulator, or the Expo app throughout development period because at the end of the day, we want it to work on mobile. I also found that the `native-web` doesn't support a lot of mobile features, which should be fine, as long as it's guaranteed to work on phones.